be/src/exec/pipeline/scan/scan_operator.cpp

```diff
@@ -41,6 +33,13 @@ ScanOperator::~ScanOperator() {
 Status ScanOperator::prepare(RuntimeState* state) {
     RETURN_IF_ERROR(SourceOperator::prepare(state));
 
+    _chunk_source_profiles.reserve(_io_tasks_per_op);
+    _is_io_task_running.reserve(_io_tasks_per_op);
+    _chunk_sources.reserve(_io_tasks_per_op);
```

Comment:
`_is_io_task_running` and `_chunk_sources` should be resized to `_io_tasks_per_op`, but `_is_io_task_running` is a `std::vector<std::atomic_bool>`, which couldn't be resized but only be initialized with a size.

Reply:
OK.